src/js/core.js

```diff
@@ -395,6 +395,9 @@ function MediumEditor(elements, options) {
         AnchorPreview: AnchorPreview
     };
 
+    MediumEditor.Util = Util;
+    MediumEditor.Selection = Selection;
```

Comment:
Should we figure out & document which methods we want to be exposed on these helper objects before we expose them?  Just trying to avoid needing new major releases as we fix up the helper methods...but maybe we don't need to worry about it.